### DISCUSSION TITLE
Calendar from the Dotnet conf demo

### DISCUSSION THREAD

**Discussion by Laftek** [dotnet/aspnetcore#52357] [Q&A] [answered]
Is there any plan to share the calendar from the demo? It looks great.

@SteveSandersonMS 

sorry for tagging you. Thanks.

**Comment by SteveSandersonMS** [comment, marked as answer]
It comes from https://github.com/SteveSandersonMS/presentation-2021-01-NDCLondon/tree/410d6c27c8b8479526507be5be4f75a281ce29de/demo/CalendarApp. Note that's a fairly old sample (.NET 5) but I was able to pull the calendar component out and use it in a .NET 8 app without trouble.

**Reply by Laftek** [reply]
Thanks again Steve!
